.travis.yml

```diff
@@ -21,6 +21,7 @@ cache:
 install: "python setup.py develop"
 
 script: 
+  - "export DASHBOARD_CONFIG=/home/travis/build/kloostert/Zeeguu-API/dashboard_test.cfg"
+  - "export DASHBOARD_LOG_DIR=/home/travis/build/kloostert/Zeeguu-API/"
```

Comment:
privacy issue? :)

Reply:
I'll look into making the paths relative instead of absolute, would that fix the privacy issue?